Description: Fix cmake to respect passed LD flags
Forwarded: Pending upstream commit
Author: Bryan Conrad <bkconrad@gmail.com>
Bug-Debian: http://bugs.debian.org/705303

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- bitfighter-019d+dfsg.orig/cmake/Platform/Linux.cmake
+++ bitfighter-019d+dfsg/cmake/Platform/Linux.cmake
@@ -6,7 +6,7 @@
 set(BF_LINK_FLAGS "-Wl,--as-needed")
 
 # Only link in what is absolutely necessary
-set(CMAKE_EXE_LINKER_FLAGS ${BF_LINK_FLAGS})
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${BF_LINK_FLAGS}")
 
 # 
 # Compiler specific flags
